device-registry/src/main/scala/org/genivi/sota/device_registry/Boot.scala

```diff
@@ -78,6 +83,20 @@ object Boot extends BootApp with Directives with BootMigrations
       new HealthResource(db, versionMap).route
     }
 
+  private def dbOp[T<: BusMessage](msg: T) = {
```

Comment:
Also I strongly object to this type, you say that `msg` can have any type as long as it have `BusMessage` as a supertype. But if you pass it anything other than an `UpdateSpec` you are screwed. So remove the `T` altogether and just take an `UpdateSpec`.

> Any customer can have a car painted any color that he wants so long as it is black.